### ISSUE TITLE
Python3 version

### ISSUE THREAD

**Issue by weirdan** [nsmgr8/vitra#9]
Debian has [switched to python3 only](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=729924) vim bindings, rendering this plugin broken.

Is there a plan to release python3-compatible version?

**Comment by nsmgr8** [comment]
Yes, python3 is in the wishlist. I myself use some hack around it to make it work on my Ubuntu machine. I'll release after I clean it up.
